jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt

```diff
@@ -601,3 +608,8 @@ data class ResponseContext(
     val sessionId: String,
     val completionType: CodewhispererCompletionType
 )
+
+interface CodeWhispererCodeCompletionServiceListener {
+    fun onSuccess() {}
+    fun onFailure() {}
```

Comment:
where is onfailure used

Reply:
oh, it hasn't been used, will remove it for now

Reply:
and add it back when needed

Reply:
sorry Richard, forgot this PR hasn't been reviewed by Andrew